SPECS/qt5-qtbase/qt5-qtbase.spec

```diff
@@ -141,6 +141,7 @@ Patch68: qtbase-everywhere-src-5.11.1-python3.patch
 # https://fedoraproject.org/wiki/Changes/Qt_Wayland_By_Default_On_Gnome
 # https://bugzilla.redhat.com/show_bug.cgi?id=1732129
 Patch80: qtbase-use-wayland-on-gnome.patch
+Patch81: CVE-2020-24742.nopatch
```

Comment:
Please add a comment explaining why we nopatch something.

Reply:
```suggestion
# CVE-2020-24742 is not applicable to v5.12.11 (already fixed in this version)
Patch81: CVE-2020-24742.nopatch
```